spec/libraries/filename_sanitizer_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe FilenameSanitizer do
+  let(:test_class) { Class.new { include FilenameSanitizer } }
+  describe 'sanitize' do
+    let(:filename) { 'lol/lol\lol:' }
+    subject { test_class.new.sanitize(filename) }
+
+    it { is_expected.to eq('lol lol lol ') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.